org.eclipse.jdt.junit/src/org/eclipse/jdt/internal/junit/ui/JUnitMessages.properties

```diff
@@ -190,6 +190,7 @@ JUnitContainerWizardPage_wizard_title=JUnit Library
 JUnitContainerWizardPage_option_junit3=JUnit 3
 JUnitContainerWizardPage_option_junit4=JUnit 4
 JUnitContainerWizardPage_option_junit5=JUnit 5
+JUnitContainerWizardPage_enableVintage=Enable Vintage Engine
```

Comment:
For a user (like me) it's not selfexplaining what "Vintage Engine" is all about. Can you please add a some explanation or link to https://junit.org/junit5/docs/current/user-guide/#migrating-from-junit4 ? 
Is it same as"JUnit 3 + JUnit 4"? That would be understandable.

Reply:
> For a user (like me) it's not selfexplaining what "Vintage Engine" is all about

I don't think there is enough space/context to explain JUnit5 concepts to users in this dialog

> Is it same as"JUnit 3 + JUnit 4"? That would be understandable.

Vintage is JUnit 5 + Support for old JUnit 3 + JUnit 4

Reply:
![grafik](https://github.com/eclipse-jdt/eclipse.jdt.ui/assets/1331477/9cac03d0-b86d-4c74-929d-6464049cffdd)

I made now a clickable link to the documentation and added a tooltip, I think there is not much more we can do.